javascript/javascript3/week3/promiseAndClass.js

```diff
@@ -0,0 +1,34 @@
+// Getting into promises
+
+async function githubRepos(repository) {
+    const fetchLink = `https://api.github.com/search/repositories?q=user:${repository}`
+    await fetch(fetchLink)
+        .then((res) => res.json())
+        .then((data) => {
+            const ul = document.querySelector('ul')
+
+            const li = document.createElement('li')
+            ul.appendChild(li)
```

Comment:
Please use some more descriptive variable names e.g. `reposotorysList` instead of `ul`

Reply:
You are right I will update that 👍